.github/workflows/build_and_test.yml

```diff
@@ -21,6 +21,29 @@ jobs:
       - run: yarn --frozen-lockfile
       - run: yarn ${{ matrix.step }}
 
+  test-playwright:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v3
+      - run: yarn --frozen-lockfile
+
+      # Cache Playwright browsers
+      - name: Cache Playwright browsers
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/ms-playwright
+          key: playwright-browsers-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            playwright-browsers-${{ runner.os }}-
+
+      # Install Playwright browsers if not cached
+      - run: yarn playwright install
+
+      - run: yarn build
```

Comment:
do we need to build?

Reply:
Scratch that, we do need it https://github.com/ar-io/arns-react/commit/8c5e27d2ba8a7e8155700c899f529cb71bd67b62